lib/config/configuration.go

```diff
@@ -1754,6 +1754,21 @@ kubernetes matchers are present`)
 				AssumeRole: assumeRole,
 			})
 		}
+		for _, azureMatcher := range fc.Discovery.AccessGraph.Azure {
+			regions := azureMatcher.Regions
+			if len(regions) == 0 {
+				return trace.BadParameter("missing regions in access_graph.azure")
```

Comment:
should we set a wildcard or error?

Reply:
A wildcard would be better. I.e. not setting regions and getting an error is not a very useful interaction, whereas not setting regions in order to see resources from all regions is much more useful.

Reply:
Added this suggestion to https://github.com/gravitational/access-graph/issues/1376